LICENSE

```diff
@@ -1,26 +1,26 @@
-Copyright YEAR      Mojca Miklavec
+Copyright (c) 2019, The MacPorts Project.
+All rights reserved.
 
-Redistribution and use in source and binary forms, with or without modification,
-are permitted provided that the following conditions are met:
-
-1. Redistributions of source code must retain the above copyright notice, this
-list of conditions and the following disclaimer.
-
-2. Redistributions in binary form must reproduce the above copyright notice,
-this list of conditions and the following disclaimer in the documentation and/or
-other materials provided with the distribution.
-
-3. Neither the name of the copyright holder nor the names of its contributors
-may be used to endorse or promote products derived from this software without
-specific prior written permission.
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions
+are met:
+1. Redistributions of source code must retain the above copyright
+   notice, this list of conditions and the following disclaimer.
+2. Redistributions in binary form must reproduce the above copyright
+   notice, this list of conditions and the following disclaimer in the
+   documentation and/or other materials provided with the distribution.
+3. Neither the name of Apple Inc., The MacPorts Project nor the
```

Comment:
Why mention Apple, to start with?

Let's just use a simple 2-clause BSD (we ended up doing that in https://github.com/macports/macports-webapp/blob/master/LICENSE, see https://github.com/macports/macports-webapp/pull/115). Base started at Apple, but there's no need to mention Apple here.

Reply:
that's a good point (I remove Apple from the copyright notice, but missed this instance...). Since all the other ```upt``` things are BSD-3 I think it would make sense to keep this the same, or...?